### DISCUSSION TITLE
How do I get rid of this log error:

### DISCUSSION THREAD

**Discussion by sanderlv** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#485] [General]
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:961
First occurred: 17:24:04 (1 occurrences)
Last logged: 17:24:04

Update of camera.xiaomi_cloud_map_extractor is taking over 10 seconds

**Comment by PiotrMachowski** [comment]
You can't really get rid of it at this moment
